Library/Homebrew/cmd/fetch.rb

```diff
@@ -253,7 +253,7 @@ def run
               end
 
               message = "#{downloadable.download_type.capitalize} #{downloadable.name}"
-              $stdout.puts "#{status} #{message}"
+              $stdout.print "#{status} #{message}#{last ? "" : "\n"}"
```

Comment:
```suggestion
              $stdout.print "#{status} #{message}#{"\n" if last}"
```

Reply:
Would be `unless` in this case, not sure that's better.

Reply:
Yeh, `unless` seems better than a ternary with `""`; nicer to rely on the default `nil` here that `unless` or `if` get you.

```suggestion
              $stdout.print "#{status} #{message}#{"\n" unless last}"
```